jest.config.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
-    "testEnvironment": "node",
-    "collectCoverage": true,
-    "collectCoverageFrom" : ["src/**/*.ts", "!src/types/**", "!src/*.ts", "!src/**/test.ts"],
+    testEnvironment: 'node',
+    collectCoverage: true,
+    collectCoverageFrom: ['src/**/*.ts', '!src/types/**'],
+    testPathIgnorePatterns: ['<rootDir>/src/__tests__/__fixtures__'],
```

Comment:
Why do you need it? I don't expect to find any tests in `__fixtures__ `.

Reply:
Jest's default test match pattern includes all js files under `__tests__` instead of only ones suffixed with `.test.js`. If we rename this directory to something else this will no longer apply and we can continue relying on suffixing with `.test.js`.